public/angular/js/services/visualization.js

```diff
@@ -0,0 +1,15 @@
+angular.module('Aggie')
+
+.factory('Visualization', function($resource) {
+	var endDate = new Date();
+	var startDate = new Date();
+	startDate.setTime(startDate.getTime() - 2 * (24 * 60 * 60 * 1000));
+	console.log("/api/v1/viz/?before=" + endDate.toISOString() + "&after=" + startDate.toISOString());
+  return $resource(

```

Comment:
There are some big spacing inconsistencies here, please convert all these tabs to spaces for consistency with the rest of the codebase (most editors allow you to set a default and easily change between them; you could also find/replace and do it all in one go).

Tabs tend to be problematic when working in teams because they render differently for everyone. Thank you!

Reply:
This should be fixed now